From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Rostyslav Lesovyi <rostyslav.lesovyi@globallogic.com>
Date: Fri, 25 Sep 2026 17:31:19 +0200
Subject: [PATCH] Remove _PLATFORM_TURRIS_ flags

Removed guard flags from the bridge-mode, _get_shell_output() and AtomBRMac.

---
 source/lm/lm_main.c    | 2 +-
 source/lm/lm_wrapper.c | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/source/lm/lm_main.c b/source/lm/lm_main.c
index 614225f..ac6506b 100644
--- a/source/lm/lm_main.c
+++ b/source/lm/lm_main.c
@@ -2953,7 +2953,7 @@ static void *Hosts_StatSyncThreadFunc(void *args)
         }
         else
         {
-#if !defined(_PLATFORM_RASPBERRYPI_) && !defined(_PLATFORM_TURRIS_) && !defined(_PLATFORM_BANANAPI_R4_)
+#if !defined(_PLATFORM_RASPBERRYPI_) && !defined(_PLATFORM_BANANAPI_R4_)
             if(bridgemode)
             {
                 Send_Eth_Host_Sync_Req(); 
diff --git a/source/lm/lm_wrapper.c b/source/lm/lm_wrapper.c
index 2fe1de1..6d012d7 100644
--- a/source/lm/lm_wrapper.c
+++ b/source/lm/lm_wrapper.c
@@ -1250,7 +1250,7 @@ RET1:
 }
 #endif
 
-#if !defined(INTEL_PUMA7) && !defined(_COSA_BCM_MIPS_) && !defined(_COSA_BCM_ARM_) && !defined(_PLATFORM_TURRIS_) && !defined(_COSA_QCA_ARM_)
+#if !defined(INTEL_PUMA7) && !defined(_COSA_BCM_MIPS_) && !defined(_COSA_BCM_ARM_) && !defined(_COSA_QCA_ARM_)
 /*
    Note that there are two versions of _get_shell_output() used with RDKB.
    This version, which accepts a char * command as the first argument, is
@@ -1287,7 +1287,7 @@ int lm_wrapper_get_arp_entries (char netName[LM_NETWORK_NAME_SIZE], int *pCount,
    // snprintf(buf, sizeof(buf), "ip nei show | grep %s | grep -v 192.168.10 > %s", netName, ARP_CACHE_FILE);
 
     // XB6/XF3 Do not have this interface. Remove constant warnings.
-#if !defined(INTEL_PUMA7) && !defined(_COSA_BCM_MIPS_) && !defined(_COSA_BCM_ARM_) && !defined(_PLATFORM_TURRIS_) && !defined(_COSA_QCA_ARM_)
+#if !defined(INTEL_PUMA7) && !defined(_COSA_BCM_MIPS_) && !defined(_COSA_BCM_ARM_) && !defined(_COSA_QCA_ARM_)
     // This is added to remove atom mac from the connected device list.
     if (pAtomBRMac[0] == '\0' || pAtomBRMac[0] == ' ') {
         fp = v_secure_popen("r","ifconfig l2sd0 | grep HWaddr | awk '{print $5}' | cut -c 1-14");
